### DISCUSSION TITLE
Will multilingual tts models be made?

### DISCUSSION THREAD

**Discussion by maximxlss** [snakers4/silero-models#230] [General]
i'm currently looking for a solution for russian tts. This has awesome performance and is easy to use and fast, but my problem with this is that it can do only one language at a time (english words are common in my application). I was thinking of just transliterating, but I also wonder if this may get something like it in the future or if you know any other tts that offers good russian performance.

**Comment by denizsincar29** [comment]
Yep, I am facing exact same problem. Want to get all english words pronounced as real english, not even вот ар ю дуинг.

**Reply by snakers4** [reply]
Hi,

This can and has been done by us, but we reserve such models for commercial applications and clients.

**Comment by denizsincar29** [comment]
if it is commercial, than how to find absolutely free and good quality neural multilingual synth with text normalization?
